x16-smc.ino

```diff
@@ -435,7 +439,7 @@ void HardReboot() {                             // This never works via I2C... W
 
 void startBootloader() {
   if (bootloaderFlags == 3 && bootloaderTimer > 0) {
-    ((void(*)(void))0x1e02)();
+    ((void(*)(void))0xf01)();
```

Comment:
What does this change do? Looks like a magic number for a function call address, maybe?

Reply:
IIRC, the discovery was that the previous code did not do the right thing because the raw address pointer needs to be in words, not in bytes.

Reply:
Yes.

Strangely enough, that change had already been committed into master. I don't know why it marked is as a change in this PR.

But yes, the previous code was calling the wrong word address.

Reply:
I see. I think we should replace it with a `#define` that tells us what the value is and maybe give a hint as to how it is calculated. I am guessing it is something like 8192 (size of flash) - 512 (size of bootloader) + 1 (for some sort of ID byte at the beginning) shifted right once. I'll open a separate issue for that since it is purely a code hygiene issue.